tests/models/language/generation/test_mistral.py

```diff
@@ -273,47 +273,44 @@ def test_mistral_function_calling(vllm_runner, model: str, dtype: str) -> None:
 
 
 @pytest.mark.parametrize("model", MODELS)
-@pytest.mark.parametrize("guided_backend",
-                         ["outlines", "lm-format-enforcer", "xgrammar"])
-def test_mistral_guided_decoding(
-    monkeypatch: pytest.MonkeyPatch,
+@pytest.mark.parametrize("structured_output_backend", ["guidance", "xgrammar"])
+def test_mistral_structured_output(
     vllm_runner,
     model: str,
-    guided_backend: str,
+    structured_output_backend: str,
 ) -> None:
-    with monkeypatch.context() as m:
-        # Guided JSON not supported in xgrammar + V1 yet
-        m.setenv("VLLM_USE_V1", "0")
-
-        with vllm_runner(
-                model,
-                dtype='bfloat16',
-                tokenizer_mode="mistral",
-                guided_decoding_backend=guided_backend,
-        ) as vllm_model:
-            guided_decoding = GuidedDecodingParams(json=SAMPLE_JSON_SCHEMA)
-            params = SamplingParams(max_tokens=512,
-                                    temperature=0.7,
-                                    guided_decoding=guided_decoding)
+    with vllm_runner(
+            model,
+            dtype='bfloat16',
+            tokenizer_mode="mistral",
+            structured_output_config=dict(backend=structured_output_backend),
+    ) as vllm_model:
+        guided_decoding = GuidedDecodingParams(json=SAMPLE_JSON_SCHEMA)
```

Comment:
Should this be renamed `StructuredOutputParams`?

Reply:
I plan to move the rename of params to a follow up PR from this